node_modules/oae-core/lhnavigation/js/lhnavigation.js

```diff
@@ -171,15 +171,6 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
                 '_': Math.random()
             };
 
-            // Although the `title` and `url` field are optional, we need to enter them as IE9 simply doesn't
-            // handle their absense very well. We cannot lose any parameters such as the `q` query string
-            // parameter as IE9 is not able to recover it from its state.
-            var title = selectedPage.title;
-            if (query) {
```

Comment:
Hmm, no. I don't actually think any of the code here needs to be deleted... We should keep `query` in `data.query`, so we should keep that part from inside the brackets, and there's no harm in keeping the title or adding the query to `loadedPath` either.